tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
     "checkJs": false,
+    "skipLibCheck": true,
```

Comment:
possible to add this config in src/v3 dir?

Reply:
Type errors on `yarn build:release` at project root without `skipLibCheck=true`:
```
$ ttsc --noEmit false --declaration --emitDeclarationOnly --declarationMap
node_modules/@types/ajv-errors/index.d.ts(6,10): error TS2614: Module '"ajv"' has no exported member 'Ajv'. Did you mean to use 'import Ajv from "ajv"' instead?
node_modules/@types/eslint-scope/node_modules/@types/eslint/helpers.d.ts(1,6): error TS2300: Duplicate identifier 'Prepend'.
node_modules/@types/eslint-scope/node_modules/@types/eslint/lib/rules/index.d.ts(10,14): error TS2300: Duplicate identifier 'Rule'.
node_modules/@types/eslint/helpers.d.ts(1,6): error TS2300: Duplicate identifier 'Prepend'.
node_modules/@types/eslint/lib/rules/index.d.ts(10,14): error TS2300: Duplicate identifier 'Rule'.
>> error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>> error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>> Exited with code: 1.
>> Error executing child process: Error: Process exited with code 1.
Warning: Task "exec:build-types" failed. Use --force to continue.

Aborted due to warnings.


Execution Time (2022-05-13 14:37:33 UTC-4)
loading tasks          593ms  ▇ 1%
exec:clean             30.7s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 70%
propertiesToJSON:main  500ms  ▇ 1%
exec:generate-config   709ms  ▇ 2%
exec:build-types       11.6s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 26%
Total 44.1s

error Command failed with exit code 3.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
re-enabled lib checks, i.e., `skipLibChecks=false`